docs/src/ref_impl/build_and_run.md

```diff
@@ -32,6 +32,7 @@ Then you can use one of the following instructions for the supported targets:
 | Generic x86 Сomputer | x86_64           | [Running Ghaf Image for x86 Computer](./build_and_run.md#running-ghaf-image-for-x86-computer) |
 | NVIDIA Jetson AGX Orin  | AArch64          | [Ghaf Image for NVIDIA Jetson Orin AGX](./build_and_run.md#ghaf-image-for-nvidia-jetson-orin-agx)     |
 | NXP i.MX 8QM-MEK        | AArch64          | [Building Ghaf Image for NXP i.MX 8QM-MEK](./build_and_run.md#building-ghaf-image-for-nxp-imx-8qm-mek)     |
+| MICROCHIP icicle-kit    | RISCV64          | [MICROCHIP icicle-kit](./build_and_run.md#building-ghaf-image-for-microchip-icicle-kit)                     |
```

Comment:
Please see my comment on a new reference configuration https://github.com/tiiuae/ghaf/pull/165#issuecomment-1581997751

Reply:
Discussed with other stakeholders and we decided to integrate Polarfire as reference platform for now.